src/libraries/System.Private.CoreLib/src/System/String.cs

```diff
@@ -295,13 +295,19 @@ private static unsafe string Ctor(ReadOnlySpan<char> value)
 
         public static string Create<TState>(int length, TState state, SpanAction<char, TState> action)
```

Comment:
```suggestion
        public static string Create<TState>(int length, TState state, [NotNull] SpanAction<char, TState> action)
```

Reply:
The parameter is already non-nullable.  Such annotation is duplicative.